tests/e2e-demos/e2e-demo.go

```diff
@@ -185,40 +183,118 @@ var _ = framework.E2ESuiteDescribe(Label("e2e-demo"), func() {
 				}
 
 				// Start to watch the pipeline until is finished
-				It(fmt.Sprintf("wait %s component %s pipeline to be finished", componentTest.Type, componentTest.Name), func() {
+				It(fmt.Sprintf("waits %s component %s pipeline to be finished", componentTest.Type, componentTest.Name), func() {
 					if componentTest.ContainerSource != "" {
 						Skip(fmt.Sprintf("component %s was imported from quay.io/docker.io source. Skipping pipelinerun check.", componentTest.Name))
 					}
 					Expect(fw.HasController.WaitForComponentPipelineToBeFinished(component.Name, application.Name, namespace)).To(Succeed(), "Failed component pipeline %v", err)
 				})
 
-				// Obtain a snapshot for the SnapshotEnvironmentBinding
-				if componentTest.ContainerSource != "" {
-					It("create snapshot for component imported from quay.io/docker.io source", func() {
-						snapshotName := SnapshotName + "-" + util.GenerateRandomString(4)
-						snapshotComponents := []appservice.SnapshotComponent{
-							{Name: component.Name, ContainerImage: component.Spec.ContainerImage},
-						}
-						snapshot, err = fw.ReleaseController.CreateSnapshot(snapshotName, namespace, application.Name, snapshotComponents)
-						Expect(err).NotTo(HaveOccurred())
-					})
-				} else {
-					It("check if the component's snapshot is created when the pipelinerun is targeted", func() {
+				It("creates a release plan and an environment", func() {
```

Comment:
It looks like the environment is actually getting created on [line 120](https://github.com/redhat-appstudio/e2e-tests/pull/309/files#diff-b565962958bd1109477d223cee4cc34ca0ccb8951570668926dea813ec3aa997R120).